src/core/statistics/useStatistics.ts

```diff
@@ -0,0 +1,52 @@
+// Useful resources for Mapp tracking:
+// https://documentation.mapp.com/1.0/en/manual-track-request-25105181.html
+// https://documentation.mapp.com/1.0/en/how-to-send-manual-tracking-requests-page-updates-7240681.html
+
+export type CustomClickParameter = {
+  [key: number]: number | { [key: number]: string };
+};
+
+export type EventData = {
+  linkId: string;
+  customClickParameter: CustomClickParameter;
+};
```

Comment:
Please rework this structure for event data. Based on [the examples provided by Mapp](https://documentation.mapp.com/1.0/en/manual-track-request-25105181.html) I see the following:

1. Neither `linkId` or `customClickParameter` are required as event data properties.
2. Many other properties are used.

[The documentation lists to so many different parameters](https://documentation.mapp.com/1.0/en/which-parameters-can-be-sent-to-mapp-intelligence-7211946.html) that I do not think it is relevant for us to limit them. To me an object with properties would be sufficiently specific.

Reply:
Good point! I generalized it to recognize customClickParameter object, but also not limit the EventData type to it.